### PR TITLE
feat/fix(#1011): rewriting Barcode Configuration Admin page in React

### LINKED ISSUE #1011: Re Write BarCode Configuration Page in the new React UI
Old Page https://oe3.openelis-global.org/api/OpenELIS-Global/BarcodeConfiguration

### PR DESCRIPTION
**Pull request by adityadeshlahre**
### fix #1011

- [x] The PR title includes a brief description of the work done, including the Issue number if applicable.
- [x] The PR includes a video showing the changes for the work done.
- [x] The PR title follows conventional commit label standards.
- [x] The changes confirm to the OpenElis Global x3 Styleguide and design documentation.
- [x] The changes include tests or are validated by existing tests.
- [x] I have read and agree to the Contributing Guidelines of this project.

## Summary
Rewriting admin page in ReactJS

#### FrontEnd
image

##### Getting perfect data from restController GET route working 
image

## Related Issue

[Add a link to the related issue or mention it here if applicable]

## Other

[Add any additional information or notes here]

### CODE REVIEW

**Comment by adityadeshlahre** [comment]
### UIUPDATE

image
image

Now working on backend [ GET & POST ] request !
By Taking use of RESTCONTROLLER

**Comment by adityadeshlahre** [comment]
### Updated WorkFlow

barCode


#### At last commit added notification forget to shoot in video !

**Comment by mozzy11** [comment]
Thanks @adityadeshlahre any blocker or progress ??

**Comment by adityadeshlahre** [comment]
> Thanks @adityadeshlahre any blocker or progress ??

Hey @mozzy11 
fixing merge conflicts and it will be ready to get merged

File: src/main/java/org/openelisglobal/barcode/controller/rest/BarcodeConfigurationRestController.java [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    @Autowired
+    private BarcodeInformationService barcodeInformationService;
+
+    @InitBinder
+    public void initBinder(WebDataBinder binder) {
+        binder.setAllowedFields(ALLOWED_FIELDS);
+    }
+
+    @GetMapping(value = "/BarcodeConfiguration")
+    // public Object showBarcodeConfiguration(HttpServletRequest request)
```
Comment (mozzy11) [inline src/main/java/org/openelisglobal/barcode/controller/rest/BarcodeConfigurationRestController.java:53]: @adityadeshlahre  , why is this whole method commented out  ??

Comment (mozzy11) [reply src/main/java/org/openelisglobal/barcode/controller/rest/BarcodeConfigurationRestController.java:53]: was this intentional or bt mistake ??

**Pull request merged**